tests/test-wrapper

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Simo sorce <simo@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+TEST_PARAMS=(${1//-/ })
+
+TEST_NAME=$(basename ${TEST_PARAMS[0]})
+TEST_PATH=$(dirname ${TEST_PARAMS[0]})
+TOKEN_DRIVER=${TEST_PARAMS[1]}
+
+if [ -f "./tmp.${TOKEN_DRIVER}/testvars" ];  then
+    source ./tmp.${TOKEN_DRIVER}/testvars
+else
+    exit 77 # token not configured, skip
+fi
+
+# some tests are compiled, others are just distributed scripts
+# so we need to check both the current tests build dir and the
+# source tests dir in the make distcheck case
```

Comment:
nit: I would adjust the comment to mention out-of-source build instead of distcheck as you can build out-of-source also outside of distcheck (but noone really does it except for `distcheck` though).